### PR TITLE
Issue #177 -- Move integration tests into integration-test module

### PR DESCRIPTION
**Pull request by VaclavDedik**
This commit moves the integration tests from ext module to integration-test module. I hope I've done it right, I replaced the maven-resources-plugin that copied the build extension into ${it.maven.home}/lib/ext wtih maven-dependency-plugin, not sure if this is the best solution but I couldn't think of any other.

### CODE REVIEW

File: integration-test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    <dependency>
+      <groupId>org.commonjava.maven.ext</groupId>
+      <artifactId>pom-manipulation-ext</artifactId>
+    </dependency>
+  </dependencies>
+
+  <profiles>
+    <profile>
+      <id>run-its</id>
+
```
Comment (rnc) [inline integration-test/pom.xml]: It might be simpler to have the profiling in the modules in the parent pom and keep this one simple.

**Comment by rnc** [comment]
Don't we also need to modify the top level pom.xml to add the new integration-test module - so we can build everything ?

**Comment by VaclavDedik** [comment]
Yes we do, and I did change the top level pom.xml.. it seems github can't show the entire diff because it is too big. You can see that the top level pom changed by looking at for example [this](https://github.com/VaclavDedik/pom-manipulation-ext/blob/master/pom.xml#L414) line in my fork.

File: integration-test/pom.xml (hunk 1 above)
Comment (VaclavDedik) [inline integration-test/pom.xml]: You are right, it is probably better, I'll move the profile.

**Comment by rnc** [comment]
@VaclavDedik : Re top pom , apologies missed that.

**Comment by rnc** [comment]
test this please

**Comment by VaclavDedik** [comment]
I made the profile modification you suggested, I can see that it makes much more sense this way.

**Pull request merged**